src/biogeochem/CNVegStructUpdateMod.F90

```diff
@@ -292,8 +293,10 @@ subroutine CNVegStructUpdate(num_soilp, filter_soilp, &
             !depth of snow required for complete burial of grasses
          endif
 
-         elai(p) = max(tlai(p)*fb, 0.0_r8)
-         esai(p) = max(tsai(p)*fb, 0.0_r8)
+         !grass and crop snow burial changes with PFT height
+         !accounts for a 20% bending factor, as used in Lombardozzi et al. (2018) GRL 45(18), 9889-9897
```

Comment:
- [ ] As in SatellitePhenologyMod, I think this comment belongs a few lines up, with the fb setting

- [ ] More importantly, it looks like you haven't fixed fb here as you did for the SatellitePhenology code. (Maybe you just haven't done that step yet.)